indy_client/test/cli/test_node.py

```diff
@@ -1,173 +0,0 @@
-import base58
-from plenum.bls.bls_crypto_factory import create_default_bls_crypto_factory
-
-from plenum.common.constants import NODE_IP, CLIENT_IP, CLIENT_PORT, NODE_PORT, \
-    ALIAS, BLS_KEY, BLS_KEY_PROOF
-from plenum.common.keygen_utils import init_bls_keys
-from plenum.common.util import randomString
-from plenum.test.cli.helper import exitFromCli
-from stp_core.network.port_dispenser import genHa
-
-from indy_client.test.cli.helper import doSendNodeCmd
-
-
-def test_add_new_node(newNodeAdded):
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
We are having pretty same functionality in 
indy-plenum/plenum/test/pool_transactions/test_nodes_data_changed.py
Please, tell me, if we need to integrate this in node